packages/dashmate/src/core/startCoreFactory.js

```diff
@@ -34,13 +34,12 @@ function startCoreFactory(
     const coreCommand = [
       'dashd',
     ];
+
     const isMasternode = config.get('core.masternode.enable');
-    const operatorKey = config.get('core.masternode.operator.privateKey');
 
-    if (options.wallet) {
-      coreCommand.push('--disablewallet=0');
-    } else if (isMasternode && Boolean(operatorKey)) {
-      coreCommand.push(`-masternodeblsprivkey=${operatorKey}`);
+    if (isMasternode) {
```

Comment:
I guess we should set operator key to null for this run if option.wallet is passed, isn't?